oahray/book-club
# #155770676 Create skeletal landing page

#### What does this PR do?
This PR adds a skeletal landing page for the app

#### Description of Task to be completed?
- Configure app structure
- Setup landing page

#### How should this be manually tested?
Run the app and visit `http://bookclub-dev.com:5000`. You should see a simple landing page interface.

#### What are the relevant pivotal tracker stories?
[#155770676](https://www.pivotaltracker.com/story/show/155770676)

#### Any background context you want to add?
N/A

#### Screenshots
N/A
